client/components/videos-ui/index.jsx

```diff
@@ -20,6 +20,9 @@ const VideosUi = ( {
 	FooterBar,
 	areVideosTranslated = true,
 	intent = undefined,
+	title = false,
+	subtitle = false,
+	options = false,
```

Comment:
Can you rename the options property to be more descriptive? `options` is too generic.

Reply:
Done, also changed title to headerTitle and headerSubtitle